editor/src/components/canvas/canvas-strategies/absolute-move-strategy.ts

```diff
@@ -117,3 +140,40 @@ function createMoveCommandsForElement(
     ['top', 'bottom', 'left', 'right'] as const,
   )
 }
+
+function snapDrag(
+  drag: CanvasPoint,
+  constrainedDragAxis: ConstrainedDragAxis | null,
+  jsxMetadata: ElementInstanceMetadataMap,
+  selectedElements: Array<ElementPath>,
+  canvasScale: number,
+): {
+  snappedDragVector: CanvasPoint
+  guidelinesWithSnappingVector: Array<GuidelineWithSnappingVector>
+} {
+  const multiselectBounds = getMultiselectBounds(jsxMetadata, selectedElements)
+
+  // This is the entry point to extend the list of snapping strategies, if we want to add more
+
+  const { snappedDragVector, guidelinesWithSnappingVector } = runLegacySnapping(
```

Comment:
There is no reason to destructure if you just recreate and return the same object again in line 167